.eslintrc.json

```diff
@@ -8,7 +8,12 @@
         "node": true
     },
     "rules": {
-        "prettier/prettier": "error",
+        "prettier/prettier": [
+            "error",
+            {
+                "endOfLine": "auto"
```

Comment:
What does this do @mufi-ibrahimjee ?

Reply:
Prettier had an issue print Delete cr with new mui. This solved it

Reply:
That is because there is no carriage return character at the end of the file which needs to be handled by us

Reply:
we should not override this settings since we are all working on different OSes. Read more [here](https://prettier.io/docs/en/options.html#end-of-line).

Reply:
i'll remove the options